### ISSUE TITLE
How to use multiple losses

### ISSUE THREAD

**Issue by liwensheng2020** [open-mmlab/mmsegmentation#485]
Hello,
       Could I use multiple losses for training? For example, I want to set the loss function to loss = 0.5*CrossEntropyLoss + 0.5*DiceLoss.

**Comment by liwensheng2020** [comment]
Oh, I find this

**Comment by DesertsP** [comment]
> Oh, I find this

Could you please share your method to use multiple losses?

**Comment by liwensheng2020** [comment]
I changed four files as follows:
https://github.com/open-mmlab/mmsegmentation/pull/244/files
